Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -179,7 +181,7 @@ class HabitTrackingController: UIViewController {
             nextButton.setImage(UIImage(named: "black-yellow-next-arrow"), for: .normal)
             titleLabel.text = "Cardio"
             descriptionLabel.text = "Daily heart pump so you can jog to your 10AM with ease"
-            habits = ["Run for 15 mins", "Play a sport for 30 mins", "Go for a walk after dinner"]
+            habits = ["Walking to class", "Walk 10000 steps per day", "Running on a treadmill for 20 minutes", "Jumprope for 5 minutes"]
```

Comment:
Can these be brought into something like `Constants.swift` for easy finding later? Will this data ever come from backend? Are these the only habits you can ever do?

Reply:
Sounds good! These are just the default/recommended habits--the users can pick among them or create their own. I don't think backend will end up have=ing these since they are static